src/backend/aspen/api/views/auth.py

```diff
@@ -146,7 +146,9 @@ async def login(
     )
 
 
-async def create_user_if_not_exists(db, userinfo) -> Tuple[User, Optional[Group]]:
+async def create_user_if_not_exists(
+    db, auth0_mgmt, userinfo
+) -> Tuple[User, Optional[Group]]:
```

Comment:
I think you need to merge in the latest from `trunk`.

Reply:
Or eliminate all the changes to users / auth, not sure how they snuck into this PR.